site/src/lib/blocks.ts

```diff
@@ -58,9 +58,9 @@ export const readBlocksFromDisk = (): ExpandedBlockMetadata[] => {
       author: metadata.packagePath.split("/")[0].replace(/^@/, ""),
       icon: getBlockMediaUrl(metadata.icon, metadata.packagePath),
       image: getBlockMediaUrl(metadata.image, metadata.packagePath),
-      lastUpdated: buildConfig.find(
-        ({ workspace }) => workspace === metadata.name,
-      )?.timestamp,
+      lastUpdated:
+        buildConfig.find(({ workspace }) => workspace === metadata.name)
+          ?.timestamp ?? null,
```

Comment:
NextJS cannot serialize `undefined` returned from `getStaticProps`, and this is called from one